Source/DafnyCore/Options/DeveloperOptionBag.cs

```diff
@@ -6,6 +6,22 @@ namespace Microsoft.Dafny;
 
 public class DeveloperOptionBag {
 
+  public static readonly Option<string> SplitPrint = new("--sprint",
```

Comment:
I slightly wonder whether these should be `--print-splits` and `--print-passive`, though I'm not sure of that. They're developer options, so maybe shorter names to type are preferable to immediately-understandable names, especially since they'd only be understandable to someone who knows how Dafny works internally even with the longer names.